web/src/layout/leftpanel/countrypanel.js

```diff
@@ -196,7 +196,7 @@ const CountryPanel = ({
   const data = useCurrentZoneData() || {};
 
   const parentPage = {
-    pathname: isMobile ? '/ranking' : '/map',
```

Comment:
Is this separate from getting it to work? I know there's an issue where it keeps going back to ranking on the mobile version

Reply:
Great!